frontend/src/components/UserMenu/AccountModal/index.jsx

```diff
@@ -143,10 +155,14 @@ export default function AccountModal({ user, hideModal }) {
               </label>
               <input
                 name="password"
-                type="password"
+                type="text"
```

Comment:
does it make sense to make this text visible as a password input?

Reply:
For this one, yes. We do not have a verify password field and I changed it to text to match all other password setting fields.